src/helpers/handlers/index.ts

```diff
@@ -11,14 +12,22 @@ declare global {
   }
   const mapboxgl: typeof mapboxgl;
 }
-
+export interface Feature {
```

Comment:
Do you think we should use the built-in `Feature` interface? i.e. `import { Feature } from 'geojson';`

Reply:
That would most preferred. I tried it out and found Feature interface does not have layer which we require in this case so I decided to use this one instead

Reply:
Should we then extend that built in feature interface, instead of creating a new one?

Reply:
yeah! most definitely